### ISSUE TITLE
Error: Cannot find module '...\node_modules\node-zopfli\lib\binding\node-v48-win32-x64\zopfli.node'

### ISSUE THREAD

**Issue by kbdguy** [pierreinglebert/node-zopfli#85]
simply running zopfli with no params, or any params, will return this error
(root folders abbreviated, here, by "...").  Windows 10 x64, node v 6.9.5 (x64)

**Comment by senaev** [comment]
I have the same problem
```
Error: Cannot find module 'C:\Users\senaev\IdeaProjects\project\node_modules\node-zopfli\lib\binding\node-v59-win32-x64\zopfli.node'
    at Function.Module._resolveFilename (module.js:557:15)
    at Function.Module._load (module.js:484:25)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\senaev\IdeaProjects\project\node_modules\node-zopfli\lib\zopfli.js:7:14)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\senaev\IdeaProjects\project\node_modules\zopfli-webpack-plugin\dist\index.js:17:19)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
error Command failed with exit code 1.
```

**Comment by sshmyg** [comment]
+1

**Comment by pierreinglebert** [comment]
Try with the new published v2.0.3
